### DISCUSSION TITLE
How to find a Memory Usage of Docker container in Rancher Desktop ?

### DISCUSSION THREAD

**Discussion by Ramaraj-Marimuthu** [rancher-sandbox/rancher-desktop#1475] [Q&A]
Do we have any options to find a states (Memory Usage, CPU Usage) of container in Rancher Desktop ? (similar to Docker Desktop)

Please share us the documentation links. It will be helpful for us.

**Comment by mattupstate** [comment]
Try running `docker stats` from the command line.
